docs/develop/persisting-add-in-state-and-settings.md

```diff
@@ -1,45 +1,151 @@
 ---
 title: Persist add-in state and settings
 description: Learn how to persist data in Office Web Add-in applications running in the stateless environment of a browser control.
-ms.date: 11/30/2023
+ms.date: 02/08/2024
 ms.localizationpriority: medium
 ---
 
 # Persist add-in state and settings
 
-[!include[information about the common API](../includes/alert-common-api-info.md)]
-
 Office Add-ins are essentially web applications running in the stateless environment of a browser iframe or a webview control. (For brevity hereafter, this article uses "browser control" to mean "browser or webview control".) When in use, your add-in may need to persist data to maintain the continuity of certain operations or features across sessions. For example, your add-in may have custom settings or other values that it needs to save and reload the next time it's initialized, such as a user's preferred view or default location. To do that, you can:
 
-- Use members of the Office JavaScript API that store data as either:
-  - Name/value pairs in a property bag stored in a location that depends on add-in type.
-  - Custom XML stored in the document.
+- [Use techniques provided by the underlying browser control](#browser-storage).
+- [Use the application-specific Office JavaScript APIs for Excel, Word, and Outlook that store data](#application-specific-settings-and-persistence).
+
+If you need to persist state across documents, such as tracking user preferences across any documents they open, you'll need to use a different approach. For example, you could use [SSO](use-sso-to-get-office-signed-in-user-token.md) to obtain the user identity, and then save the user ID and their settings to an online database.
+
+## Browser storage
+
+Persist data across add-in instances with tools from the underlying browser control, such as browser cookies or HTML5 web storage ([localStorage](https://developer.mozilla.org/docs/Web/API/Window/localStorage) or [sessionStorage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage)).
+
+Some browsers or the user's browser settings may block browser-based storage techniques. You should test for availability as documented in [Using the Web Storage API](https://developer.mozilla.org/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API).
+
+### Storage partitioning
+
+As a best practice, any private data should be stored in partitioned `localStorage`. [Office.context.partitionKey](/javascript/api/office/office.context#office-office-context-partitionkey-member) provides a local storage partition key only accessible by your add-in. The following example shows how to use the partition key with `localStorage`. Note that the partition key is undefined in environments without partitioning, such as the browser controls for Windows applications.
```

Comment:
Re: "a local storage partition key only accessible by your add-in"
Strictly, the local Storage (not the key) is only accessible by add-in. A different add-in will get the same key, but when using it in the context of an add-in, only that add-in can access it.
Can we use similar wording like Chrome local storage below like "This means that data stored by storage APIs, such as local storage, are only available to contexts with the same origin and the same top-level site.", 

Not to specific details, can we say something like this here? "This means that data stored by storage APIs, such as local storage, are only available to the same use context."